lib/checkout_sdk/api_client.rb

```diff
@@ -64,6 +65,7 @@ def invoke(method, path, authorization, body = nil, idempotency_key = nil, param
       json_body = CheckoutSdk::JsonSerializer.to_custom_hash(body).to_json
 
       begin
+        @log.info "#{method}: /#{path}"
```

Comment:
What do you think about logging the body here?

Reply:
Like we discussed, this probably is not a good idea atm, because there are some PII on the request sometimes, so we could wait until any merchant request this as debug